tests/admin_docs/tests.py

```diff
@@ -123,6 +123,15 @@ def test_missing_docutils(self):
         finally:
             utils.docutils_is_available = True
 
+    def test_simplify_regex(self):
```

Comment:
Let's add a separate test class for the new test method (use the one I added) like `AdminDocViewFunctionTests` since the existing tests run twice, the second time with multiple template engines, but that's not needed for this.